### ISSUE TITLE
mitmproxy crash on Server Side Replay

### ISSUE THREAD

**Issue by krzysiekbielicki** [mitmproxy/mitmproxy#481]
after logging some requests in mitmproxy I pressed S for server-side replay, a/t for replay and mitmproxy crashed with error:
```
Traceback (most recent call last):
  File "/usr/lib64/python2.7/site-packages/libmproxy/console/__init__.py", line 614, in run
    self.ui.run_wrapper(self.loop)
  File "/usr/lib64/python2.7/site-packages/urwid/raw_display.py", line 272, in run_wrapper
    return fn()
  File "/usr/lib64/python2.7/site-packages/libmproxy/console/__init__.py", line 839, in loop
    self.prompt_execute(k)
  File "/usr/lib64/python2.7/site-packages/libmproxy/console/__init__.py", line 766, in prompt_execute
    msg = p(txt, *args)
  File "/usr/lib64/python2.7/site-packages/libmproxy/console/flowlist.py", line 124, in server_replay_prompt
    self.master.options.replay_ignore_params, self.master.options.replay_ignore_content
TypeError: start_server_playback() takes exactly 9 arguments (8 given)
```

**Comment by krzysiekbielicki** [comment]
it seems to be resolved in latest master

**Comment by veb** [comment]
I can't replicate it on the latest master either.

**Comment by mhils** [comment]
I think this issue is already fixed by #447. As a workaround, you can upgrade to master - this will definitely be fixed in the next release.

Cheers,
Max
